packages/frontend/src/embed/EmbedViewer.vue

```diff
@@ -319,26 +186,51 @@ body::-webkit-scrollbar {
   height: 100vh !important;
   width: 100vw !important;
   position: fixed;
+
   &::-webkit-scrollbar {
     display: none;
   }
+
+  top: 0;
```

Comment:
These originally came from SpeckleViewer.vue. Since it's now asynchronized, we need to copy these styles over